CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+# Contribution Guidelines
+
+Anyone is welcome to contribute to this project. Feel free to get in touch with other community
+members on [Matrix][1], the mailing list or through issues here on GitHub.
+
+## Code of Conduct
+
+You must agree to abide by the [Mozilla Community Participation Guidelines][2]
+
+## Bug Reports
+
+You can file issues here on GitHub. Please try to include as much information as you can and under
+what conditions you saw the issue.
+
+## Sending Pull Requests
+
+Patches should be submitted as pull requests (PR).
+
+Before submitting a PR:
+
+- Ensure you are pulling from the most recent `main` branch.
+- Your code must run and pass all the automated tests before you submit your PR for review.
+  "Work in progress" or "Draft" pull requests are allowed to be submitted, but should be clearly
```

Comment:
Reading this statement conveys that a DRAFT PR can be merged if labeled so. AFAIK a Draft PR should never be merged. It should always be converted to an Open PR to be able to be merged. Isn't that so?

Reply:
When I see this guideline elsewhere and knowing the GitHub UI context, I interpret 'submitted' to be a synonym of 'opened' or 'created'. I'm happy to consider other wording, do you have a suggestion?